src/logic/calculate.js

```diff
@@ -0,0 +1,43 @@
+import operate from './operate'
+
+export function calculate(calculator, buttonName) {
```

Comment:
This cant be done. Because of "[array].includes(buttonName)" returns a boolean and the switch statement that I am comparing to the cases is actually a string